Ij article tweaks

thewtex opened the pull request:
@jbvimort Outstanding work! Here are a few suggested tweaks to the IJ article.

thewtex committed:
DOC: Correct rendering of Francois

The accent drops the c in the rendered PDF.

thewtex committed:
DOC: itkTextureFeature -> itkTextureFeatures

thewtex committed:
DOC: Fix trailing whitespace

thewtex committed:
DOC: Article grammatical tweaks

dzenanz commented on doc/itkTextureFeature.tex:
```diff
@@ -49,7 +49,7 @@
 % At minimum, give your name and an email address.  You can include a
 % snail-mail address if you like.
-\author{Jean-Baptiste Vimort$^{1}$, Matthew McCormick$^{1}$ François Budin$^{1}$ and Beatriz Paniagua$^{1}$}
+\author{Jean-Baptiste Vimort$^{1}$, Matthew McCormick$^{1}$ Francois Budin$^{1}$ and Beatriz Paniagua$^{1}$}
```
Instead of this, use `\usepackage[utf8]{inputenc}` in the header?

thewtex replied:
@dzenanz Thanks for the pointer!

I tested it, and it worked -- I will update the branch.

thewtex committed:
DOC: Use accent in Francois

Revert "DOC: Correct rendering of Francois"

This reverts commit f905103237fad828928946cec0d024142cfbd730.

Add \usepackage[utf8]{inputenc}

Suggested-by: Dženan Zukić <dzenan.zukic@kitware.com>

jbvimort commented:
Thank you so much for all those corrections! :)

I will merge it right away!
